pkg/cmd/set/set_env.go

```diff
@@ -134,10 +135,16 @@ type EnvOptions struct {
 	enforceNamespace       bool
 	clientset              *kubernetes.Clientset
 	resRef                 api.ResourceRef
+	cfg					   *rest.Config
 
 	genericclioptions.IOStreams
 }
 
+type control struct {
```

Comment:
No need `control struct`，in https://github.com/openkruise/kruise-tools/pull/19  I have removed it.